gnovm/tests/files/recover12a.gno

```diff
@@ -0,0 +1,26 @@
+package main
+
+func anotherRecover() {
+	if r := recover(); r != nil {
+		println(r)
```

Comment:
```sh
$ go run ./gnovm/tests/files/recover12a.gno.gen.go 
(0x45c540,0x47e008)
```

The result is again different in go, you need to convert `r` before printing it to make it similar:

```suggestion
		println(r.(string))
```

Reply:
Is this necessary for this PR? It seems to be due to a difference in the `println` implementation, so it may be out of scope; the tests pass as written.

Reply:
Yes you're probably right, I'll discard my comments related to that.

Reply:
I can't resolve them, let's just ignore them ^^